dll/win32/uxtheme/nonclient.c

```diff
@@ -252,10 +245,9 @@ void ThemeCalculateCaptionButtonsPos(HWND hWnd, HTHEME htheme)
 
     GetThemePartSize(htheme, NULL, iPartId, 0, NULL, TS_MIN, &ButtonSize);
 
-    ButtonHeight = GetSystemMetrics( wi.dwExStyle & WS_EX_TOOLWINDOW ? SM_CYSMSIZE : SM_CYSIZE);
-    ButtonWidth = MulDiv(ButtonSize.cx, ButtonHeight, ButtonSize.cy);
+    ButtonWidth = MulDiv(ButtonSize.cx, buttonHeight, ButtonSize.cy);
 
-    ButtonHeight -= 4;
+    ButtonHeight = buttonHeight - 4;
```

Comment:
Maybe call that new variable a bit different, so you don't have to play with the fire (same names with different casing).

Reply:
Good call. Fixed.